src/redux/data.js

```diff
@@ -134,9 +133,18 @@ const resolve = (request, body, payload) => {
   if (Array.isArray(data)) {
     records = records.concat(data);
     ids = records.map(({ id }) => id);
-  } else {
+  } else if (data) {
     records = [data];
     ids = [data.id];
+  } else {
```

Comment:
Instead of this additional `else`, lets remove the previous `if (data)` addtion and make `data` default to `{ id: package.params.id }`.

Something like this above the first `if` statement should work:
```js
if (!data && request.params.id) {
  data = { id: request.params.id };
}
```

Reply:
On second thought, let's keep `else if (data)` for sanity. But still create the default `data` above the first `if` statement

Reply:
Nice!